templates/admin/base_site.html

```diff
@@ -0,0 +1,435 @@
+{% extends "admin/base_site.html" %} {% block extrahead %}
+
+<link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/normalize.css@8.0.1/normalize.css"
+  crossorigin="anonymous"
+/>
+
+<style>
+  /* GLOBAL */
+
+  html {
+    /* COLOR PALLETTE */
+    --color-blue: #1717e5;
+
+    --color-black: #000;
+    --color-white: #fff;
+
+    --color-warning: #ff7d0d;
+    --color-error: #ff0d4f;
+    --color-border: rgba(255, 255, 255, 0.3);
+
+    /* TYPOGRAPHY */
+    --font-family-standard: "IBM Plex Sans", sans-serif;
+    --font-family-mono: "IBM Plex Mono", "Courier New", Courier, monospace;
+
+    --font-size-orange: 32px;
+    --line-height-orange: 44px;
+
+    --font-size-apple: 24px;
+    --line-height-apple: 36px;
+
+    --font-size-peach: 16px;
+    --line-height-peach: 24px;
+
+    /* SPACING */
+    --page-max-width: 1140px;
+    --padding-standard: 16px;
+
+    /* OTHER */
+    --border-radius-standard: 4px;
+  }
+
+  * {
+    box-sizing: border-box;
+  }
+
+  *::before,
+  *::after {
+    box-sizing: inherit;
+    font-family: inherit;
+  }
+
+  body {
+    color: var(--color-white);
+    background: radial-gradient(
+      circle at center,
+      var(--color-blue) -40%,
+      var(--color-black) 75%
+    );
+    background-attachment: fixed;
+    line-height: 1.65;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  body.login {
+    background: radial-gradient(
+      circle at center,
+      var(--color-blue) -40%,
+      var(--color-black) 75%
+    );
+  }
+
+  body *,
+  body *::before,
+  body *::after {
+    font-family: inherit;
+    font-weight: inherit;
+    color: inherit !important;
+    background-color: inherit !important;
+  }
+
+  .warning {
+    color: var(--color-media-watch) !important;
+    font-size: var(--font-size-peach) !important;
+  }
+
+  .help {
+    padding-left: 0 !important;
+    padding-right: 0 !important;
+  }
+
+  .errorlist {
+    color: var(--color-error) !important;
+  }
+
+  /* PAGE LAYOUT */
+  #container {
+    max-width: var(--page-max-width);
+    margin: 0 auto;
+  }
+
+  /* HEADER */
+  #header {
+    color: var(--color-white);
+    background-color: var(--color-black);
+    padding: 48px 32px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  #branding #site-name {
+    padding: 0;
+    margin: 0;
+    font-weight: bold;
+    font-size: 16px;
+  }
+
+  #branding #site-name a {
+    display: block;
+    padding: 4px 14px;
+    margin: 0;
+    padding: 4px 14px;
+    border: 2px solid var(--color-white);
```

Comment:
Because of branding reasons we should remove the border here.

Reply:
Done